cmd/_integration-tests/transport/http_test.go

```diff
@@ -606,7 +607,21 @@ func testHTTP(
 		t.Fatal(errors.Wrap(err, "cannot make http request"))
 	}
 
-	err = json.Unmarshal(respBytes, &resp)
+	switch v := resp.(type) {
+	case *pb.GetWithQueryResponse:
```

Comment:
Yeah, I found that out the hard way - tried combining cases, still got an interface.  It's more verbose (and repetitive) than I'd like, but as `jsonpb.Unmarshal` wants a `proto.Message`, this was the shortest / simplest way I could find to do it.